src/common/SPELLS/monk.ts

```diff
@@ -163,6 +163,11 @@ const spells = spellIndexableList({
     name: "Invoker's Delight",
     icon: 'inv_inscription_80_warscroll_battleshout',
   },
+  FURY_OF_XUEN_BUFF: {
```

Comment:
May as well add the stacking buff SPELL for this while you're in here
`https://www.wowhead.com/spell=396167`

Reply:
Good idea.

Random question: do you have and tips for tracking down spell ids for different buffs / spells? I often don't have much confidence just from searching through wowhead, instead resorting to just going in game and getting the buff the proc myself.

Reply:
![image](https://user-images.githubusercontent.com/1672786/236548460-8e7adee7-14a8-48e1-bd92-3ed469d8bd60.png)

If you open a report in WoWA, you can click "About" and then "View all events" to see a nice view of all of the events in a log - you can filter from there as well.

![image](https://user-images.githubusercontent.com/1672786/236548614-b4321661-ba71-473a-bc9a-64c6e48d3552.png)